src/MeetingsSDKAdapter.test.js

```diff
@@ -211,6 +212,46 @@ describe('Meetings SDK Adapter', () => {
     });
   });
 
+  describe('fetchMeetingTitle()', () => {
+    beforeEach(() => {
+      global.console = {
+        warn: jest.fn(),
+      };
+    });
+
+    test('returns meeting sipUri as title', async () => {
+      const meetingTitle = 'sipUri';
+
+      expect(await meetingSDKAdapter.fetchMeetingTitle(meetingTitle)).toEqual(meetingTitle);
+    });
+
+    test('returns person displayName if the destination is a personID', async () => {
+      const meetingTitle = 'PEOPLE';
+
+      meetingSDKAdapter.datasource.people.get = jest.fn(() => Promise.resolve({displayName: 'displayName'}));
+
+      expect(await meetingSDKAdapter.fetchMeetingTitle(meetingTitle)).toEqual("displayName's Personal Space");
+    });
+
+    test("returns room's title if the destination is a personID", async () => {
+      const meetingTitle = 'ROOM';
```

Comment:
Can you rename for clarity?
```suggestion
      const meetingTitle = 'HYDRA_ROOM_ID';
```